### PR TITLE
voicemask can select speech verb

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
title

## Why / Balance
prevents metagaming: Pets-The-Fish a well-known lizard has moth speech verbs? KOS all moths!

accents wyci

## Technical details
uses SpeechVerb from 2 month ago pr i forgor about :trollface:

draft until the state handling selection works and doesnt select None every time idk why it does it but the verb used is correct

## Media

https://github.com/space-wizards/space-station-14/assets/39013340/25de8ec9-d2df-4c3a-ab6a-af36d28b2648

list is properly sorted now:

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
SpeechVerbPrototype now has required `Name` locid, shown in voice mask dropdown

**Changelog**
:cl:
- tweak: Voicemasks can now mimic speech verbs, like flutters or growls.

### CODE REVIEW

**Comment by UbaserB** [comment]
what about screaming, is screaming sounds changed too?

**Comment by deltanedas** [comment]
its just speech verb not emote sounds, in that case someone trying to meta a lizard-moth can just notice the wings.

**Comment by UbaserB** [comment]
> its just speech verb not emote sounds, in that case someone trying to meta a lizard-moth can just notice the wings.

once i was a moth impersonating a human CE and no one noticed i was a moth, for the whole shift. i was only outed as a moth once i screamed lol

File: Content.Shared/VoiceMask/SharedVoiceMaskSystem.cs [hunk 1]
```diff
@@ -11,9 +11,10 @@ public enum VoiceMaskUIKey : byte
...
 public sealed class VoiceMaskBuiState : BoundUserInterfaceState
 {
-    public string Name { get; }
+    public readonly string Name;
+    public readonly string? Verb;
 
-    public VoiceMaskBuiState(string name)
+    public VoiceMaskBuiState(string name, string? verb)
     {
         Name = name;
```
[the remark this answers is not in the document]
Comment (deltanedas) [reply Content.Shared/VoiceMask/SharedVoiceMaskSystem.cs:19]: guh

File: Content.Client/VoiceMask/VoiceMaskNameChangeWindow.xaml [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
+    <BoxContainer Orientation="Vertical" Margin="5">
         <Label Text="{Loc 'voice-mask-name-change-info'}" />
-        <BoxContainer Orientation="Horizontal">
+        <BoxContainer Orientation="Horizontal" Margin="5">
             <LineEdit Name="NameSelector" HorizontalExpand="True" />
             <Button Name="NameSelectorSet" Text="{Loc 'voice-mask-name-change-set'}" />
         </BoxContainer>
+        <BoxContainer Orientation="Horizontal" Margin="5">
+        	<Label Text="{Loc 'voice-mask-name-change-speech-style'}" />
+	        <OptionButton Name="SpeechVerbSelector" /> <!-- Populated in LoadVerbs -->
```
Comment (metalgearsloth) [inline Content.Client/VoiceMask/VoiceMaskNameChangeWindow.xaml:13]:
```suggestion
            <Label Text="{Loc 'voice-mask-name-change-speech-style'}" />
            <OptionButton Name="SpeechVerbSelector" /> <!-- Populated in LoadVerbs -->
```

**Pull request merged**